.eslintrc.js

```diff
@@ -159,42 +173,48 @@ module.exports = {
     //"no-sync": "off",
 
     //# Stylistic Issues
+    "array-bracket-newline": ["warning", "consistent"],
     "array-bracket-spacing": ["error", "never"],
+    "array-element-newline": ["warning", "consistent"],
     "block-spacing": ["warn", "always"],
     "brace-style": ["error", "1tbs", { "allowSingleLine": true }],
     "camelcase": ["error", { "properties": "never" }],
+    //"capitalized-comments": "off",
     "comma-dangle": ["warn", "always-multiline"],
     "comma-spacing": ["error", { "before": false, "after": true }],
     "comma-style": ["error", "last"],
     "computed-property-spacing": ["error", "never"],
     "consistent-this": ["warn", "self"],
     "eol-last": "warn",
     "func-call-spacing": ["error", "never"],
+    //"func-name-matching": "off",
     //"func-names": "off",
     //"func-style": "off",
+    "function-paren-newline": ["warning", "consistent"],
     //"id-blacklist": "off",
     //"id-length": "off",
     //"id-match": "off",
+    "implicit-arrow-linebreak": ["error", "beside"],
     "indent": ["error", 4, { "SwitchCase": 1 }],
     //"jsx-quotes": "off",
     "key-spacing": ["error", { "beforeColon": false, "afterColon": true }],
     "keyword-spacing": ["warn", { "before": true, "after": true }],
     //"line-comment-position": "off",
     "linebreak-style": "error",
     //"lines-around-comment": "off",
-    "lines-around-directive": ["warn", { "before": "never", "after": "always" }],
+    "lines-between-class-members": "off",
     "max-depth": ["error", 4],
     //"max-len": "off",
     //"max-lines": "off",
+    //"max-lines-per-function": "off",
```

Comment:
@GianlucaGuarini doesn't make it sense to set this to around 100 lines per function? So we avoid huge function that have to be maintained

Reply:
I agree but then this config will break probably 50% of our internal tools 😄

Reply:
Sure, we can do that :D